torchvision/transforms/v2/functional/_color.py

```diff
@@ -83,30 +103,15 @@ def adjust_brightness_image_tensor(image: torch.Tensor, brightness_factor: float
     return output if fp else output.to(image.dtype)
 
 
-adjust_brightness_image_pil = _FP.adjust_brightness
+def adjust_brightness_image_pil(image: PIL.Image.Image, brightness_factor: float) -> PIL.Image.Image:
```

Comment:
Needed this to test the PIL dispatch. Can be factored out with the rest of the test changes.

Reply:
> Is it because we have changed the tests?

It is because our newly added test for `adjust_brightness` needs a proper function to test the dispatch rather than an alias.

> We still don't (yet) have registered PIL kernels right?

Correct.